update/update.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+# Input variables
+LOCAL_NCBI_DIR=$(pwd)/ncbi
+PREVIOUS_UTA_VERSION=uta_20210129b
+
+# 1. Download files for SeqRepo and UTA updates
+# Input: N/A
+# Output: LOCAL_NCBI_DIR populated with downloaded files
+mkdir -p $LOCAL_NCBI_DIR
+docker build --file Dockerfile.download --tag ncbi:latest --progress plain .
+docker run -it --mount type=bind,source=$LOCAL_NCBI_DIR,target=/data/ncbi ncbi:latest
+
+# 2. Prepare SeqRepo
+# Input: TBD
+# Output: TBD
+
+# 3. Prepare UTA
+docker build --file ../misc/docker/uta.dockerfile --tag uta:$PREVIOUS_UTA_VERSION --build-arg uta_version=$PREVIOUS_UTA_VERSION ../misc/docker
```

Comment:
Why not just pull the image from biocommons and run?

Reply:
I needed to rebuild the image when I edited the pg_hba.conf in the local dockerfile, but now that I undid the pg_hba.conf change, I can change it back.

Reply:
Done in #9